esp/templates/survey/survey_form.html

```diff
@@ -22,20 +22,24 @@
                 {% for sec in sections %}
                 <tr>
                     <td class="underline" align="center">{{ sec }}</td>
-                    <td class="underline" align="center">{% if not sec.started %}<span style="color: grey">Not available</span>{% elif sec.completed %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?sec={{ sec.id }}">Take Survey</a>{% endif %}</td>
+                    <td class="underline" align="center">{% if not sec.started %}<span style="color: grey">Not available</span>{% elif sec.completed %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?{% if survey_id %}survey_id={{ survey_id }}&{% endif %}sec={{ sec.id }}">Take Survey</a>{% endif %}</td>
                 </tr>
                 {% endfor %}
             {% endif %}
             {% if general_survey %}
                 <tr>
                     <td class="underline" align="center">General Program Survey</td>
-                    <td class="underline" align="center">{% if not general_available %}<span style="color: grey">Not available</span>{% elif general_done %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?general">Take Survey</a>{% endif %}</td>
+                    <td class="underline" align="center">{% if not general_available %}<span style="color: grey">Not available</span>{% elif general_done %}<span style="color: green">Completed</span>{% else %}<a class="btn fancybutton no-dec" href="{{ request.path }}?{% if survey_id %}survey_id={{ survey_id }}&{% endif %}general">Take Survey</a>{% endif %}</td>
                 </tr>
             {% endif %}
         </table>
+        <br><br>
     {% else %}
         <p class="message">This survey does not have any questions.</p>
     {% endif %}
+    {% if student_results %}
+        <p class="message">You can view the results of the student survey(s) for your class(es) by <a href="{{ request.path }}/review">going here</a>.
```

Comment:
If there are multiple teacher surveys, then this shows up only after you click on one of the teacher surveys, which is weird to me since the student survey is not connected to the teacher surveys. So at the very least I think it should show up on the page with the list of multiple surveys instead of when you click one of them. 

However, I've realized since my original review that I would prefer if the survey results were linked in the "Manage your classes" boxes at the bottom of the techerreg page instead. Since there's a box per class and that box should contain everything about the class, I think that's the most logical place for a teacher to find their survey results. I think I would just add a button "View survey results" under the "Email ___ Students" button (if survey results exist).

Reply:
(you can change that last condition to be just if a student survey exists. If there are no results I'm okay with having the link be there and say "no results yet! check back later" when clicked)